src/App.tsx

```diff
@@ -1,61 +1,114 @@
+import { FormEvent, useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+const getUser = (userId: number) => {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const completeTodos: Todo[] = todosFromServer.map(todo => ({
```

Comment:
This name might be missleading
```suggestion
const todosWithUsers: Todo[] = todosFromServer.map(todo => ({
```